src/ContMechTensors.jl

```diff
@@ -2,6 +2,11 @@ __precompile__()
 
 module ContMechTensors
 
+import Base.@pure
+
+using StaticArrays
+import StaticArrays.setindex
```

Comment:
This is not in the tagged version of `StaticArrays`. Need to add the new tag (when its tagged) to `REQUIRE` later